docs/getting_started/user-guide.rst

```diff
@@ -861,15 +861,15 @@ Add System Modules to a View
 ----------------------------
 
 Views have support for customization, such as a system module that you always want loaded.
-We do this by way of an extra view_module that is generated in the root of the view (and
+We do this by way of an extra ``.view_<view>`` that is generated in the root of the view (and
 always attempted to be loaded) by the installed modules. For example, let's say that when
 we load a view module named mpi, we always want to load a system module named "openmpi" and "mymod." We could do:
 
 .. code-block:: console
 
     $ shpc view add <view> system_modules <name1> <name2>
     $ shpc view add mpi system_modules openmpi mymod
-    Wrote updated .view_module: /home/vanessa/Desktop/Code/shpc/views/mpi/.view_module
+    Wrote updated .view_mpi: /home/vanessa/Desktop/Code/shpc/views/mpi/.view_mpi.lua
```

Comment:
So the logic here is to have more than one view_ file in this subdirectory? What is the use case for that again (sorry been a while since we discussed!)

I think the reason this doesn't work is because we have to have logic to source the file in whatever modulefile we are symlinking, and we cannot know the name in advance.

Reply:
See other comment thread below :)